pyproject.toml

```diff
@@ -109,7 +109,7 @@ databricks-sdk = { version = "^0.24.0", optional = true }
 databricks-sql-connector = { version = "^3.1.1", optional = true }
 fastapi = { version =  "^0.96.0", optional = true }  # Need to bump pydantic to 2.X.X
 feast = { version = "^0.35.0", optional = true, extras = ["snowflake", "redis", "mysql"]}
-featurebyte-freeware = { version = "^0.2.16", optional = true }
+featurebyte-freeware = { version = "^0.2.18", optional = true }
```

Comment:
@kchua78 @bushwhackr From the error, seems this commit (https://github.com/featurebyte/freeware/commit/fc96716582edaccfe4fe4fc3736d0fae32e35443) is not included. Should we bump the freeware to higher version?